qemu/tests/ioeventfd.py

```diff
@@ -75,7 +75,7 @@ def _set_ioeventfd_options():
             dev_id = 'image1'
         elif params['drive_format'] == 'scsi-hd':
             params['bus_extra_params_image1'] = ioeventfd
-            dev_id = 'virtio_scsi_pci0'
+            dev_id = params.get('dev_id', 'virtio_scsi_pci0')
```

Comment:
Will virtio_blk not be affected?

Reply:
according to the test result, it seems no conflict with virtio_blk:
JOB ID     : d979a78cc261d53e8e387fd0b804db9d3c32ebf4
JOB LOG    : /root/avocado/job-results/job-2021-03-19T05.26-d979a78/job.log
 (1/2) Host_RHEL.m8.u4.product_av.nographic.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.4.0.s390x.io-github-autotest-qemu.unattended_install.cdrom.extra_cdrom_ks.default_install.aio_threads.s390-virtio: PASS (873.17 s)
 (2/2) Host_RHEL.m8.u4.product_av.nographic.qcow2.virtio_blk.up.virtio_net.Guest.RHEL.8.4.0.s390x.io-github-autotest-qemu.ioeventfd.check_property.s390-virtio: PASS (49.14 s)
RESULTS    : PASS 2 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /root/avocado/job-results/job-2021-03-19T05.26-d979a78/results.html
JOB TIME   : 922.97 s